__init__.py

```diff
@@ -120,17 +120,17 @@ def contentdir(self):
         return os.path.join('features', self.name())
 
 
-class SpicyAnalyzer(zeekpkg.template.Feature):
-    """Feature for a Spicy-based analyzer."""
+class SpicyProtocolAnalyzer(zeekpkg.template.Feature):
+    """Feature for a Spicy-based protocol analyzer."""
     def name(self):
-        return 'spicy-analyzer'
+        return 'spicy-protocol-analyzer'
 
     def contentdir(self):
         return os.path.join('features', self.name())
 
     def needed_user_vars(self):
         """Specify required user variables."""
-        return ['name', 'namespace']
+        return ['name', 'namespace', 'analyzer', 'unit']
```

Comment:
If you don't need `namespace` in the `spicy-analyzer` feature, you should be able to remove it here, and remove the check for `ns` in `validate()` below.

Reply:
Ah, didn't realize this isn't used anywhere here anymore now, will remove.